### DISCUSSION TITLE
Support for Intel compiler on windows

### DISCUSSION THREAD

**Discussion by DanielRichtmann** [ccache/ccache#1092] [Development]
Hi there,

first of all thank you for this great tool. I have seen a factor of 6-8x reduction in build time for our codebase on windows with it.

When using MSVC everything worked out of the box for me. For ccache to work with the intel compiler on windows I needed to make some changes to its sources. I could send a PR if there is interest from your side on upstreaming this?

**Comment by jrosdahl** [comment]
> I could send a PR if there is interest from your side on upstreaming this?

Sure, that wouldn't hurt.
